apps/browser/lib/hooks/useEvo.ts

```diff
@@ -164,9 +163,7 @@ export function useEvo({
         setError(e.message);
       }
     })();
-  }, [
-    localOpenAiApiKey
-  ]);
+  }, [localOpenAiApiKey, userWorkspace]);
```

Comment:
Seems like this PR may be dependent up on this refactor: https://github.com/polywrap/evo.ninja/issues/569